core/src/main/java/com/linecorp/armeria/server/AbstractServiceBindingBuilder.java

```diff
@@ -437,19 +215,9 @@ AbstractServiceBindingBuilder decorator(Function<T, R> decorator) {
     abstract void serviceConfigBuilder(ServiceConfigBuilder serviceConfigBuilder);
 
     final void build0(Service<HttpRequest, HttpResponse> service) {
-        if (defaultRouteBuilder != null) {
-            addRouteBuilder(defaultRouteBuilder, methods);
-        }
-        checkState(!routeBuilders.isEmpty(),
-                   "Should set a path that the service is bound to before calling this.");
-
-        for (Entry<RouteBuilder, Set<HttpMethod>> entry : routeBuilders.entrySet()) {
-            final Route route = entry.getKey()
-                                     .methods(entry.getValue())
```

Comment:
Note this line, which is not using the `methods` field.

Reply:
Yes, but `methods` field is not a class field. It is the value of `Entry`.
I think the variable name make you confuse.

Reply:
Oops! Yeah it's indeed the variable name that confused me - can we use a different one, like `routeMethods`? It's confusing to shadow the class field of the same name.

Reply:
I really appreciate your detail review. 😀 I will rename the variable to `routeMethods`.
Sorry to confuse you. 😅